### PR TITLE
doc --rpc-http-max-active-connections option

### LINKED ISSUE #630: Doc --rpc-http-max-active-connections option
PR: https://github.com/hyperledger/besu/pull/1996

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [x] you fixed all the issues raised by the tests, if any.
- [x] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

Add documentation for the --rpc-http-max-active-connection option

## Issue fixed

fixes #630 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

## Testing

https://hyperledger-besu--638.org.readthedocs.build/en/638/

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -2617,6 +2617,34 @@ To allow remote connections, set to `0.0.0.0`
...
+    BESU_RPC_HTTP_MAX_ACTIVE_CONNECTIONS=100
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    rpc-http-max-active-connections="100"
+    ```
+
+The maximum number of HTTP connections allowed for JSON-RPC. Once this limit is reached, incoming connections will be rejected. The default is 80.
```
Comment (bgravenorst) [inline docs/Reference/CLI/CLI-Syntax.md:2646]: Minor comment: Try and rephrase without "will". It's one of the words on our do not use list :)

**Review by bgravenorst — CHANGES_REQUESTED** [review]
Minor suggestions. Also, similar to the other issue  I'm not sure quotes ("") are required around integers in the config file. But we seem pretty inconsistent in the docs.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -2617,6 +2617,34 @@ To allow remote connections, set to `0.0.0.0`
...
+=== "Environment Variable"
+
+    ```bash
+    BESU_RPC_HTTP_MAX_ACTIVE_CONNECTIONS=100
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    rpc-http-max-active-connections="100"
```
Comment (NicolasMassart) [inline docs/Reference/CLI/CLI-Syntax.md:2643]: the TOML syntax for integers is indeed without quotes, see https://toml.io/en/

File: docs/Reference/CLI/CLI-Syntax.md [hunk 3]
```diff
@@ -2617,6 +2617,34 @@ To allow remote connections, set to `0.0.0.0`
...
+
+=== "Environment Variable"
+
+    ```bash
+    BESU_RPC_HTTP_MAX_ACTIVE_CONNECTIONS=100
+    ```
+
+=== "Configuration File"
+
+    ```bash
```
Comment (NicolasMassart) [inline docs/Reference/CLI/CLI-Syntax.md:2642]:
```suggestion
    ```toml
```

**Review by NicolasMassart — APPROVED** [review]

**Review by bgravenorst — APPROVED** [review]
Looks good to me.

**Pull request merged**